client/src/service/game/game.ts

```diff
@@ -1,6 +1,6 @@
 import { Game, GameStatusKind } from "../../types/game/game.types";
 import { Player } from "../../types/player/player.types";
-import { Card } from "../../types/card/card.types";
+import { Card, STAT_NAME, StatName } from "../../types/card/card.types";
```

Comment:
StatName on line 3 currently showing as unused

Reply:
Removed now, ta! I've also fixed the issue you messaged on slack about the number of cards being returned.